x11-packages/qt5-qtwebengine/build.sh

```diff
@@ -6,7 +6,7 @@ TERMUX_PKG_VERSION="5.15.12"
 TERMUX_PKG_REVISION=3
 TERMUX_PKG_SRCURL=git+https://github.com/qt/qtwebengine
 TERMUX_PKG_GIT_BRANCH=v$TERMUX_PKG_VERSION-lts
-TERMUX_PKG_DEPENDS="fontconfig, dbus, libc++, libjpeg-turbo, libminizip, libnss, libpng, libre2, libsnappy, libvpx, libwebp, libx11, libxml2, libxslt, libxkbfile, qt5-qtbase, qt5-qtdeclarative, zlib"
+TERMUX_PKG_DEPENDS="fontconfig, dbus, libc++, libdrm, libjpeg-turbo, libminizip, libnss, libpng, libre2, libsnappy, libvpx, libwebp, libx11, libxml2, libxslt, libxkbfile, qt5-qtbase, qt5-qtdeclarative, zlib"
```

Comment:
This is run-time dependencies, not build dependencies as in the subject.

Reply:
I have no idea whether `libdrm` should be specified as runtime or build dependencies. `libdrm` should be linked against with the chromium binary, but it seems that this doesn't happen on qtwebengine.